### DISCUSSION TITLE
.Net6 compatibility

### DISCUSSION THREAD

**Discussion by edgaralenpwn** [Live-Charts/LiveCharts2#523] [Ideas]
I pulled the code and was able to quickly and easily add .Net6 compatibility, and rebuild my old UWP app as a WinUI3 app using the .Net6 version I created.
Is there plans to include .Net6 support in the near future?

**Comment by beto-rodriguez** [comment]
it is supported, what's the error message you are getting?

**Reply by edgaralenpwn** [reply]
I was not able to add it to a .Net6 project. It showed as .Net5 compatible.

**Reply by beto-rodriguez** [reply]
what was the error message? how did you install the package?
